MAUI/Calendar/getting-started.md

```diff
@@ -0,0 +1,154 @@
+
+## Creating an application using the .NET MAUI Calendar
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfCalendar to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Calendar and then install it.
+
+3. To initialize the control, import the control namespace `Syncfusion.Maui.Calendar` in XAML or C# code.
+
+4. Initialize `SfCalendar.`
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3 5" %}
+
+<ContentPage   
+    . . .
+    xmlns:calendar="clr-namespace:Syncfusion.Maui.Calendar;assembly=Syncfusion.Maui.Calendar">
+
+    <calendar:SfCalendar />
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1 9 10" %}
+
+using Syncfusion.Maui.Calendar;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfCalendar calendar = new SfCalendar();
+        this.Content = calendar;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+using Syncfusion.Maui.Core.Hosting;
+namespace GettingStarted
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+
+            builder.ConfigureSyncfusionCore();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Change different calendar views
+
+The .NET MAUI Calendar control provides four different types of views to display dates and it can be assigned to the control by using the `View` property. The control is assigned to the Month view by default. Initially, all Calendar views will show the current date.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Month;
+
+{% endhighlight %}
+{% endtabs %}
+
+![change-different-calendar-views-in-maui-calendar](images/getting-started/maui-calendar-month-view.png)
+
+## Change first day of week
+
+The Calendar control rendered with `Sunday` as first day of week and it  provides customization to change the first day of the week with the FirstDayOfWeek property in month view.
```

Comment:
The Calendar control is rendered with `Sunday` as the first day of the week and it allows customization to change the first day of the week using the `FirstDayOfWeek` property in month view.

Reply:
Content modified.